readme.md

```diff
@@ -153,11 +237,31 @@ $ npm -v
 
 * **Your Answer:**
 
+No, because the package is installed as an npm module, and is in the `package.json` file.
+
+To require the package, in `index.js`, add:
+```
+const moment = require('moment')
+moment().format();
+```
```

Comment:
It's actually in _both_ the `package.json` and the `node_modules/` folder. Node looks inside of `node_modules/` to try and find `moment`.